groups/unifiedmanager/ec2.tf

```diff
@@ -10,7 +10,7 @@ module "unified_manager_ec2_security_group" {
   vpc_id      = data.aws_vpc.vpc.id
 
 
-  ingress_cidr_blocks = local.admin_cidrs
+  ingress_prefix_list_ids = local.admin_cidrs
```

Comment:
The use of `local.admin_cidrs` makes no sense here now. The value for the `ingress_prefix_list_ids` will not be CIDRs, it'll be a list of prefix list IDs.